packages/vc-verification/src/verifier/revocation-verification.ts

```diff
@@ -205,7 +205,7 @@ export class RevocationVerification {
             issuer = rolePayload?.iss as string;
             role = issuers.roleName;
             credentialStatus = rolePayload?.credentialStatus;
-            if (rolePayload?.exp && rolePayload?.exp < Date.now()) {
+            if (rolePayload?.exp && rolePayload?.exp * 1000 < Date.now()) {
```

Comment:
Optional chaining can be skipped here
```suggestion
            if (rolePayload.exp && rolePayload.exp * 1000 < Date.now()) {
```

Reply:
It throws error `object possibly undefined`